libraries/AP_FETtecOneWire/AP_FETtecOneWire.cpp

```diff
@@ -110,29 +110,31 @@ void AP_FETtecOneWire::update()
     TelemetryData t {};
     int16_t centi_erpm;
     uint16_t tx_err_count;
+    uint8_t tlm_ok = 0; //decode_single_esc_telemetry returns 1 if telemetry is ok, 0 if its waiting and 2 if there is a crc mismatch.
+    uint8_t tlm_from_id = 0;

```

Comment:
```suggestion
    uint8_t tlm_from_id;

```
It makes no sense to init this variable.